src/index.js

```diff
@@ -0,0 +1,91 @@
+import homeFood from './assets/images/home-food.jpg';
+import menuPage from './menu';
+import aboutPage from './about';
+
+const body = document.getElementById('content');
+
+function navigation() {
```

Comment:
Kindly use the ES6 arrow function instead of the "function" keyword

Reply:
@Nkuria Job well done...
The previous reviewer pointed out the use of ES6 arrow function instead of using the keyword `function`.
kindly attend to this... 👍